var programsList = [

    {
    "Name" : "PALMA",
    "Date" : "2016",
    "CoAuthors" : "Afef Cherni, Emilie Chouzenoux",
    "Technology" : "python",
    "Description" : "Processing of DOSY experiment with a MaxEnt / L1 parrallel algorithm.",
    "Diffusion" : "freely available - <a href='http://palma.labo.igbmc.fr' title='PALMA'>see web site</a>",
    "Publication" : ["arxiv 1608.07055"]
    },
    {
    "Name" : "SPIKE",
    "Date" : "2015",
    "CoAuthors" : "L.Chiron, M-A Coutouly, Jean-Philippe Starck, Christian Rolando",
    "Technology" : "python",
    "Description" : "Spectrometry Processing Innovative KErnel.",
    "Diffusion" : "freely available - <a href='https://bitbucket.org/delsuc/spike' title='SPIKE'>see web site</a>",
    "Publication" : ["arxiv 1608.06777"]
    },
    {
    "Name" : "urQRd",
    "Date" : "2013",
    "CoAuthors" : "L.Chiron",
    "Technology" : "python",
    "Description" : "Fast Denoising of large experimental datasets",
    "Diffusion" : "freely available - <a href='http://urqrd.igbmc.fr' title='urQRd'>see web site</a>",
    "Publication" : ["Efficient denoising algorithms for large experimental datasets and their applications in Fourier transform ion cyclotron resonance mass spectrometry.", 
        "Chiron, L., van Agthoven, M. A., Kieffer, B., Rolando, C. Delsuc, M.-A (2014)", 
        "Proc Natl Acad Sci USA", 
        "doi:10.1073/pnas.1306700111"] ,
    "Impact" : "International Patent"
    },
    {
    "Name" : "RamaDA",
    "Date" : "2011",
    "CoAuthors" : "M.Tanty",
    "Technology" : "python",
    "Description" : "Conformationnal analysis of protein backbone",
    "Diffusion" : "freely available - <a href='http://ramada.u-strasbg.fr' title='RamaDA'>see web site</a>",
    "Publication" : ["RamaDA:complete and automated conformational overview of proteins M. Tanty and M.-A. Delsuc", 
        "2011", 
        "Arxiv", 
        "1111.5586", 
        "http://arxiv.org/abs/1111.5586"]
    },
    {
    "Name" : "Sophora",
    "Date" : "2009",
    "CoAuthors" : "O. Assemat, M.-A. Coutouly, R.Hajjar",
    "Technology" : "python",
    "Description" : "Correction program for DOSY experiments, improves the quality of the spectra.",
    "Diffusion" : "<a href='http://www.nmrtec.com' title='Sophora'>distributed by NMRtec</a>.",
    "Publication" : ["O. Assemat, M.-A. Coutouly, R.Hajjar, and M.-A. Delsuc", 
        "2010", 
        "Validation of molecular mass measurements by means of Diffusion-ordered NMR spectroscopy; application to oligosaccharides.",
        "CRChimie",
        "13",
        "412-415"] ,
    "Impact" : "about 50 laboratories today"
    },
    {
    "Name" : "DOSYm 2.0",
    "Date" : "2009",
    "CoAuthors" : "M-A Coutouly; in collaboration with NMRtec",
    "Technology" : "F77 - python - Java",
    "Description" : "A 'plug-in' for TopSpin, based on NPK allowing the DOSY processing by Inverse Laplace transform.",
    "Diffusion" : "<a href='http://www.nmrtec.com' title='DOSYm'>distributed by NMRtec</a>.",
    "Impact" : "about 50 laboratories today"
    },
    {
    "Name" : "MW",
    "Date" : "2008",
    "Technology" : "Web site in python",
    "Description" : "This page implements the estimate of molecular mass from the measurement of 'Diffusion' coefficient",
    "Diffusion" : "freely available - <a href='http://abcis.cbs.cnrs.fr/htbin-post/MW/MW.py' title='MW'>see web site</a>",
    "Publication" : ["Augé et al.", 
        "2009",
        "NMR Measure of Translational Diffusion and Fractal Dimension. Application to Molecular Mass Measurement.", 
        "J Phys Chem B",
        "113",
        "1914-1918"],
    "Impact" : "26 citations (July 2015)"
    },
    {
    "Name" : "Pathways",
    "Date" : "2008",
    "Technology" : "python",
    "Description" : "coherence transfert pathways for NMR experiments with Pulsed Field Gradients",
    "Diffusion" : "from the authors",
    "Publication" : ["S. Balayssac, M.-A. Delsuc, V. Gilard, Y. Prigent, and M. Malet-Martino",
        "2009",
        "Two-dimensional DOSY experiment with excitation sculpting water suppression for the analysis of natural and biological media",
        "J.Magn.Reson.", 
        "196",
        "78–83"]    
    },
    {
    "Name" : "<i>NMRNoteBook</i>",
    "Date" : "2006",
    "CoAuthors" : "In collaboration with NMRtec",
    "Technology" : "F77 - python - Java / multiplateform",
    "Description" : "The companion program of NPK, takes care of the graphical user interface and data managment",
    "Diffusion" : "<a href='http://www.nmrtec.com' title='MW'>distributed by NMRtec</a>.",
    "Impact" : "more than 1000 users at this "
    },
    {
    "Name" : "NPK",
    "Date" : "2006",
    "CoAuthors" : "Tramesel, D., V. Catherinot; in collaboration with NMRtec",
    "Technology" : "F77 - python - Java / multiplateform",
    "Description" : "Complete NMR processing program. Allows unattended automatic processing of 1D; 2D; 3D and DOSY. Includes FT, MaxEnt and Inverse Laplace transform.",
    "Diffusion" : "freely available - <a href='http://abcis.cbs.cnrs.fr/NPK' title='NPK'>see web site</a>",
    "Publication" : ["Tramesel, D., V. Catherinot and M.-A. Delsuc",
        "2007",
        "Modeling of NMR processing, toward efficient unattended processing of NMR experiments.",
        "J.Magn.Reson.",
        "188", 
        "p56-67"],
    "Impact" : "about 50 laboratories today + embedded in NMRNoteBook"
    },
    {
    "Name" : "CRAACK",
    "Date" : "2005",
    "CoAuthors" : "C. Benod and J.L. Pons",
    "Technology" : "Web site in perl",
    "Description" : "This is an help to statistically assigning a protein spin system from the chemical shift values. CRAACK is based on two consensus algorithm which tries to acccurate the responses of existant typing modules. The first algorithm accomplishes the analysis with SVM technologie . The second one use a classical ballot algorithm.",
    "Diffusion" : "freely available - <a href='http://abcis.cbs.cnrs.fr/craack/' title='CRAACK'>see web site</a>",
    "Publication" : ["CRAACK:consensus program for NMR amino acid type assignment.",
        "Benod, C., M. A. Delsuc and J. L. Pons (2006)",
        "J Chem Inf Model",
        "46",
        "1517-22"],
    "Impact" : "~10 hits/day"
    },
    {
    "Name" : "Rio-Grande",
    "Date" : "2004",
    "Technology" : "Web site in PHP and Spip",
    "Description" : "A collective laboratory notebook, allowing the follow-up of multi-tems projects. Generates public reports as well as a private comprehensive 'Description' of the project, available to participating people.",
    "Diffusion" : "freely available - distributed on request",
    "Impact" : "Widely used in the C.B.S. lab., distributed once to a another lab."
    },
    {
    "Name" : "DOSYm 1.0",
    "Date" : "2004",
    "CoAuthors" : "Tramesel, D., T. Gostan; in collaboration with NMRtec",
    "Technology" : "F77 - Java / multiplateform",
    "Description" : "A 'plug-in' for TopSpin, based on Gifa V5 allowing the DOSY processing by Inverse Laplace transform.",
    "Diffusion" : "distributed by NMRtec and Bruker.",
    "Impact" : "about 40 laboratories today"
    },
    {
    "Name" : "RESCUE-2",
    "Date" : "2004",
    "CoAuthors" : "Thérèse Malliavin, Antoine Marin, P.Nicolas",
    "Technology" : "Web site in perl",
    "Description" : "Prediction of protein assignment from <sup>1</sup>H <sup>15</sup>N and <sup>13</sup>C chemical shifts.",
    "Diffusion" : "freely available - <a href='http://abcis.cbs.cnrs.fr/rescue2/' title='RESCUE-2'>see web site</a>",
    "Publication" : ["Marin, A., T. E. Malliavin, P.Nicolas and M. A. Delsuc", 
        "2004", 
        "From NMR chemical shifts to amino acid types:investigation of the predictive power carried by nuclei.", 
        "J.Bio.NMR", 
        "30", 
        "47-60"] ,
    "Impact" : "~10 hits/day"
    },
    {
    "Name" : "NMRb",
    "Date" : "2004",
    "CoAuthors" : "J-L.Pons  T.E.Malliavin, D.Tramesel",
    "Technology" : "Web site in PHP",
    "Description" : "NMR raw dataset repository",
    "Diffusion" : "freely available - <a href='http://nmrb.cbs.cnrs.fr/' title='NMRb'>see web site</a>",
    "Publication" : ["Pons, J. L., T. E. Malliavin, D. Tramesel and M. A. Delsuc", 
        "2004", 
        "NMRb : A web-site repository for raw NMR data-sets.", 
        "Bioinformatics", 
        "20", 
        "3707-3709"]
    },
    {
    "Name" : "Gifa V5.0",
    "Date" : "2002",
    "CoAuthors" : "D.Tramesel, in collaboration with NMRtec",
    "Technology" : "F77 - C - Java / multiplateform",
    "Description" : "A rewrite of Gifa v4.",
    "Diffusion" : "Commercial 'Diffusion' by NMRtec",
    "Impact" : "several dozens of licenses sold"
    },
    {
    "Name" : "TCLand",
    "Date" : "2000",
    "CoAuthors" : "M.Guillet",
    "Technology" : "matlab",
    "Description" : "Analyse et visualisation des données de PCR quantitatives",
    "Diffusion" : "copyright INSERM",
    "Publication" : ["Sebille, F., K. Gagne, M. Guillet, M.-A. Delsuc and J.-P. Soulillou", 
        "2001", 
        "Comparison of specificity and magnitude of T cell, mobilization after xenogeneic or allogeneic direct recognition.", 
        "XENOTRANSPLANTATION", 
        "8"] ,
    "Impact" : "International Patent"
    },
    {
    "Name" : "Fire",
    "Date" : "1999",
    "CoAuthors" : "T.Malliavin",
    "Technology" : "F77 and macro gifa V4",
    "Description" : "A software allowing  protein structures evaluation from a 3D 15N HSQC-NOESY spectra, with no assignment required",
    "Diffusion" : "copyright INSERM",
    "Publication" : ["Malliavin, T. A., P. Barthe and M. A. Delsuc.", 
        "2001", 
        "FIRE : an automatic protocol for protein fold recognition and NMR spectral assignment.", 
        "Theor. Chem. Acc", 
        "106", 
        "91-97"]
    },
    {
    "Name" : "RESCUE",
    "Date" : "1998",
    "CoAuthors" : "J-L Pons",
    "Technology" : "Matlab and perl",
    "Description" : "Protein assignment prediction from <sup>1</sup>H.",
    "Diffusion" : "<a href='http://bioserv.cbs.cnrs.fr/rescue/'>website</a>",
    "Publication" : ["Pons, J. L. and M.-A. Delsuc", 
        "1999", 
        "Rescue : an Artificial Neural Network tool for helping the NMR spectral assignment of proteins.", 
        "J. Bio. NMR", 
        "15", 
        "15-26"]
    },
    {
    "Name" : "Gifa assignment module",
    "Date" : "1997",
    "CoAuthors" : "T.Malliavin and J-L Pons",
    "Technology" : "Gifa v4 macro language",
    "Description" : "Assignment module for proteins and peptides, integrated into Gifa. 2D and 3D homo and heteronuclears, integration, relaxation measurments",
    "Diffusion" : "freely distributed with Gifa V4.",
    "Publication" : ["Malliavin, T. E., J. L. Pons and M. A. Delsuc", 
        "1998", 
        "An NMR assignment module implemented in the Gifa NMR processing program.", 
        "Bioinformatics", 
        "14(7)", 
        "624-631"]
    },
    {
    "Name" : "CROWD",
    "Date" : "1995",
    "CoAuthors" : "T.Malliavin",
    "Technology" : "F77 - C / Unix",
    "Description" : "Relaxation modelisation for arbitrary spin system (up to 1000 spins). Handle NOESY and ROESY experiments.",
    "Diffusion" : "freely available - <a href='http://bioserv.cbs.cnrs.fr/CROWD/'>see web site</a>"
    },
    {
    "Name" : "Gifa V4.0",
    "Date" : "1995",
    "CoAuthors" : "T.Malliavin and J-L Pons",
    "Technology" : "F77 - C / Unix",
    "Description" : "Complete NMR processing program, comes with programmable graphic interface. Comprehensive manual.",
    "Diffusion" : "<a href='https://www.github.com/delsuc/Gifa'>Github</a>",
    "Publication" : ["Pons, J. L., T. E. Malliavin and M. A. Delsuc", 
        "1996", 
        "Gifa V 4:A complete package for NMR data set processing.", 
        "J. Biomol. NMR", 
        "8(4)", 
        "445-452"] ,
    "Impact" : "Over 250 laboratories world-wide, 259 citations (January 2021)"
    },
    {
    "Name" : "Gifa V1.0 - V2.0 - V3.0",
    "Date" : "1989 - 1990 - 1993",
    "CoAuthors" : "V.Stoven, M.Robin, T.Malliavin, D.Roux,",
    "Technology" : "F77 - C / Unix - VMS",
    "Description" : "Successive versions of the Gifa NMR processing program <ul> <li>V1 : initial version, includes MaxEnt, and FT of 1D, 2D, 3D</li> <li>V2 : ported to Unix / X11-Motif - new memory management allowing data-sets of any size. - LP, baseline correction</li> <li>V3 : included an embedded macro language</li> </ul>",
    "Diffusion" : "Commercial 'Diffusion', several dozens of laboratory equipped., academical and pharmaceutical. V3 : Sold by Oxford Molecular"
    },
    {
    "Name" : "LP",
    "Date" : "1987",
    "CoAuthors" : "Feng Ni & George C. Levy",
    "Technology" : "FORTRAN 77 / Vax VMS",
    "Description" : "Complete Linear Prdiction module, implementated within the MEM/LP program Un module complet de prédiction linéaire implanté dans le logiciel MEM/LP diffusé par la société NMRi.",
    "Diffusion" : "Commercial by the NMRi company - 50 pages manual",
    "Publication" : ["Delsuc, M. A., F. Ni and G. C. Levy", 
        "1987", 
        "Improvement of Linear Prediction Processing of very Low Signal-to-Noise NMR Spectra.", 
        "J. Magn. Reson.", 
        "73", 
        "548-552"] ,
    "Impact" : "Several hundreds"
    },
    {
    "Name" : "BOWMAN",
    "Date" : "1986",
    "CoAuthors" : "D.Piveteau",
    "Technology" : "Pascal / Aspect 2000-3000",
    "Description" : "A program computing the density matrix evolution for 2 coupled spin 1/2. Displays the magnetisation trajectory in physical space. Simulates complex pulse sequences, and generates 1D and 2D FIDs, which can be processed on the spectrometer.",
    "Diffusion" : "ABACUS",
    "Publication" : ["Piveteau, D., M. A. Delsuc and J. Y. Lallemand", 
        "1986", 
        "BOWMAN A Versatile Simulation Program for High-Resolution, NMR Multipulse Experiments.", 
        "J. Magn. Reson.", 
        "70", 
        "290-294"]
    },
    {
    "Name" : "FORTH-24",
    "Date" : "1985",
    "Technology" : "Aspect 2000 -3000 Assembling 'Technology', FORTH 'Technology'",
    "Description" : "Implementation of the FORTH programming langugae on the Bruker Aspect architecture. Complete working environment : editor; copmiler, assembler, file system, real time managment. Allows driving the pulse programmer and writing pulse sequences on the WM architecture",
    "Diffusion" : "ABACUS - 30 pages Manual"
    }
];
